### PR TITLE
BP-397_REST_Routes

### PR DESCRIPTION
**Pull request by julianweise**

### CODE REVIEW

File: src/GridView/CardGrid.js [hunk 1]
```diff
@@ -1,32 +1,54 @@
...
+			);
+		}
 	}
 }
 
-export default CardGrid;
\ No newline at end of file
+export default connect(props => ({
+	lazyChildEntities: attributeList => ({
+		childEntities: `/entity/${props.currentEntity.Id}/children/type/${props.childEntityType.Id}/${attributeList}`
```
Comment (friedow) [inline src/GridView/CardGrid.js]: I would suggest renaming childEntities to entities.

Comment (friedow) [inline src/GridView/CardGrid.js]: I would suggest renaming childEntityTypes to entityTypes.

File: src/GridView/GridView.js [hunk 2]
```diff
@@ -1,32 +1,54 @@
...
-	componentDidMount() {
-		this.props.datasource.getEntities(3, this.getEntities);
-	}
 
-	getEntities(entities) {
-		this.setState({
-			entities: entities
+class GridView extends Component {
+	
+	getChildEntityTypes(parentEntityTypeId, hierarchy) {
```
Comment (friedow) [inline src/GridView/GridView.js:13]: I would suggest renaming getChildEntityTypes to getEntityTypes.

**Review by friedow — CHANGES_REQUESTED** [review]

File: src/GridView/GridView.js (hunk 2 above)
Comment (MaximilianV) [reply src/GridView/GridView.js:13]: I do see your point concerning the "child" names in CardGrid. But in GridView we should stick to this name, to make clear that we are talking about the types of child elements.

Comment (julianweise) [reply src/GridView/GridView.js:13]: Because this appears in the context of GridView it makes sense to explicitly mention the child-parent dependency.

Comment (friedow) [reply src/GridView/GridView.js:13]: You're correct, my bad.

**Review by friedow — APPROVED** [review]

**Pull request merged**